docs/reference/datatiers.asciidoc

```diff
@@ -10,8 +10,8 @@ typically share the same hardware profile:
 and hold your most recent, most-frequently-accessed data.
 * <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently
 and rarely needs to be updated.
-* <<cold-tier, Cold tier>> nodes hold time series data that is accessed infrequently and not normally updated.
-* <<frozen-tier, Frozen tier>> nodes hold time series data that is accessed rarely and never updated, kept in searchable snapshots.
+* <<cold-tier, Cold tier>> nodes hold time series data that is accessed infrequently and not normally updated. <<fully-mounted,Fully mounted indices>> of <<ilm-searchable-snapshot,{search-snaps}>> may be kept on the cold tier, eliminating the need for replicas with approximately 50% storage reduction compared to the warm tier.
```

Comment:
Great suggestion. Could you check if my latest commit satisfies the above?

I made quite a lot of modifications in the cold tier paragraph. Mainly to point out that one can use cold tier just like warm tier, or use searchable snapshot. 

```
Cold tier can be used for storing rarely updated time series data similar to 
the warm tier.  It can also be used for storing read-only
<<fully-mounted,fully mounted indices>> of
<<ilm-searchable-snapshot,{search-snaps}>>.  When one chooses to use searchable
snapshot on the cold tier, it eliminates the need for replicas with
approximately 50% storage reduction compared to the warm tier.
```

This is indeed a frequent question users are confused about the cold tier.